vendor/github.com/OpenBazaar/wallet-interface/datastore.go

```diff
@@ -32,6 +37,21 @@ func (c *CoinType) String() string {
 	}
 }
 
+func (c *CoinType) CurrencyCode() string {
+	switch *c {
+	case Bitcoin:
+		return "BTC"
+	case BitcoinCash:
+		return "BCH"
+	case Zcash:
+		return "ZEC"
+	case Litecoin:
+		return "LTC"
+	default:
+		return ""
+	}
+}
```

Comment:
That's a  good point. I think the answer is no as the `coinType` only specifies the specific coin whereas there are other things ... like the `Params()` and `CurrencyCode()` interface methods that can show if it's testnet or not.

Reply:
I would really like to make sure we're consistent with our usage and representations everywhere. I know there are places where we DO express the cointype in terms of "TBTC" or similar. It may not be of consequence to any of our logic immediately, but it would good to represent this accurately if possible. This choice also requires us to use SQL AND Marshalling to inspect the serializedContract for the real type (testing vs mainnet) instead of just being able to `where coin = "TBTC"` on a query.

Reply:
The only tables that are getting a column added are those used by the wallets. So purchases, sales, etc (which would have a serializedContract) are unaffected.